### PR TITLE
Fix saltify tracebacks

### PR DESCRIPTION
**Pull request by cameron**
- KeyError: 'saltify.get_configured_provider'
- TypeError: deploy_script() got an unexpected keyword argument 'ssh_keyfile'

### CODE REVIEW

**Comment by techhat** [comment]
You're teh awesome, @cameronboehmer!

File: saltcloud/clouds/saltify.py [hunk 1]
```diff
@@ -165,3 +165,14 @@ def script(vm_):
...
         )
     )
+
+def get_configured_provider():
+    '''
+    Return the first configured instance.
+    '''
+    return config.is_provider_configured(
+        __opts__,
+        __active_provider_name__ or 'aws',
```
Comment (s0undt3ch) [inline saltcloud/clouds/saltify.py:175]: aws?

Comment (s0undt3ch) [inline saltcloud/clouds/saltify.py:175]: make that `saltify`

Comment (techhat) [inline saltcloud/clouds/saltify.py:175]: Curse my reading too fast!

Comment (techhat) [inline saltcloud/clouds/saltify.py:175]: Fixed in 649998c23d7921808bfe57ec7a04cfa8f71aa1ac

Comment (cameron) [inline saltcloud/clouds/saltify.py:175]: sweet! yeah missed that after adding this function in a few too many places..

**Comment by cameron** [comment]
soo... can i ask a dumb question? how did the saltify provider get so broken? no tests?

**Comment by s0undt3ch** [comment]
Not a dumb question at all. Salt cloud's unit test suite is only weeks old and is in need of some love, but time is scarce... ;)

**Pull request merged**